eclass/autotools.eclass

```diff
@@ -136,6 +136,12 @@ unset _automake_atom _autoconf_atom
 # Additional options to pass to automake during
 # eautoreconf call.
 
+# @ECLASS-VARIABLE: AT_NOEAUTOHEADER
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Don't run eautoheader command if set to 'yes'; only used to workaround
```

Comment:
That noun can be written as `workaround`, but verb is `work around`.

Reply:
Thanks. Fixed.